docs/agents/cryomech_cpa1114.rst

```diff
@@ -0,0 +1,68 @@
+.. highlight:: rst
+
+.. _cryomech_cpa1114:
+
+======================
+Cryomech CPA1114 Agent
```

Comment:
Should we consider renaming this before it gets distributed too widely? It works fine with our CPA1110, so the name should probably be more general. It's a minor thing, so maybe not worth the hassle.

Reply:
Yeah, I sort of agree it should be more generally named. Are the CPA1114 and CPA1110 the only compressors in SO?

Renaming isn't too bad within the repo/ocs. Renaming the dockerhub image doesn't seem possible. However a new one can be created and the old one can be updated to explain that there's a newer image elsewhere (and eventually deleted.)

Reply:
Hmm, I'm not sure about the compressors used in SO. I'll ping the cryo channel. The old/new image names sounds a bit confusing, but if the image is really only in use at a few institutions right now maybe it won't be a huge issue.